packages/jest-runner/src/runTest.ts

```diff
@@ -186,9 +186,17 @@ async function runTestInternal(
     const esm = runtime.unstable_shouldLoadAsEsm(path);
 
     if (esm) {
-      await runtime.unstable_importModule(path);
+      const {default: setupModule} = await runtime.unstable_importModule(path);
+      // const setupModule = await import(path);
+      // console.log(`setupModule: ${JSON.stringify(setupModule)}`);
```

Comment:
@SimenB What I do wrong here? `await import()` works as expected, `console` prints the imported object. With `unstable_importModule` all I get is `undefined`. Without destructuring, it returns simply `{}`. Whatever I try to pass to the import, result is the same. Was wrestling with it for some time. I need a hint. Is this something obvious, I can’t get? (;

Reply:
Or is `unstable_importModule` not returning anything on purpose? The return type is `Promise<void>`, I was assuming this is incorrect type and that might be my mistake.

Reply:
Wait.. If I got it right, `unstable_importModule` is just evaluating the code. It does not return exports. So this means that it is not possible to support this feature in ESM. Or?